__tests__/payload.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pull_request": {
+      "number": 10
```

Comment:
What does `"number": 10` mean here?

Reply:
Random number, I can't add comments in JSON